README.rst

```diff
@@ -71,25 +71,59 @@ Next, the most basic use is to use the shortcut function, like this:
 ::
 
     >>> from premailer import transform
-    >>> print transform("""
+    >>> print(transform("""
     ...         <html>
     ...         <style type="text/css">
     ...         h1 { border:1px solid black }
     ...         p { color:red;}
     ...         p::first-letter { float:left; }
     ...         </style>
+    ...         <style type="text/css" data-premailer="ignore">
+    ...         h1 { color:blue; }
+    ...         </style>
     ...         <h1 style="font-weight:bolder">Peter</h1>
     ...         <p>Hej</p>
     ...         </html>
-    ... """)
+    ... """, base_url=None))
```

Comment:
Why did you put this one in?

Reply:
Idea here was to signal people that `transform` accepts this keyword argument in case they didn't make it further down (similar to the inclusion of `data-premailer="ignore"`). I'm not married to the idea if you don't like it, though. :-)

Incidentally (and a little off topic) is there a reason that `transform` doesn't just accept `**kwargs` and then pass it along to the Premailer constructor? I was surprised it supported `base_url` but none of the other options.

Reply:
Wow! It doesn't?! That's nuts. Can you fix it please?

The only reason the `transform` function exists at all is because it looks slightly simpler than:

```python
instance = Premailer(html)
output = instance.transform()
```